<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EscojerMasculino</class>
 <widget class="QDialog" name="EscojerMasculino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_atrasgenero">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>680</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NextPolos">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_head_5">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>530</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adultoshow">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Adulto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LegsShort">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>330</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Artefactos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ninhoshow">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ni√±o</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>280</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Polos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_BodyVest">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>130</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphics_Avatar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Body">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Foots">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>530</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapatos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NextFoots">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>270</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Short</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_head">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NextShort">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NextHead">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;- -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_jovenshow">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Joven</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Guardar">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>680</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
